Add createNamedFromBuilder to AbstractController

redheness opened the pull request:
This new method allow the user to use the `createNamedBuilder()` method from the controller

| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | symfony/symfony-docs/form/form_customization.rst#ChangingtheFormName

[Controller] Add createNamedFromBuilder() method

redheness committed:
Add createNamedFromBuilder to AbstractController

This new method allow the user to use the `createNamedBuilder()` method from the controller

redheness committed:
Update file formatting of AbsractController after previous commit

redheness committed:
Add myself to contributors

nicolas-grekas commented on CONTRIBUTORS.md:
```diff
@@ -2659,3 +2659,4 @@ Symfony is the result of the work of many people who made the code better
  - Vladislav Vlastovskiy (vlastv)
  - RENAUDIN Xavier (xorrox)
  - Yannick Vanhaeren (yvh)
+ - Kilian Riou (redheness)
```
this file is autogenerated, please revert this line

nicolas-grekas reviewed (commented):
can you please add a test and a line in the changelog of the bundle?

redheness committed:
Add test

redheness committed:
Revert "Add myself to contributors"

This reverts commit eea1f3bdd792685ae94ae7d86adf39159eb7e89f.

redheness committed:
Update changelog

ro0NL commented:
oh, actually i thought this added `createNamedForm` :) im for adding both then :+1: 

we still inject FormFactory for classic GET forms, to do  `createNamed('', KnownType::class)`.

redheness committed:
Apply patch from fabbot

https://fabbot.io/report/symfony/symfony/38145/172d7e058c3ffe495bafa1452ed9bed044c5866a

redheness committed:
Merge pull request #1 from symfony/master

Integrate symfony/symfony/master into redheness/symfony/master

redheness committed:
Merge branch 'master' into patch-1
